build-system-tests/templates/components/react/vite/template-tsconfig-vite-latest.json

```diff
@@ -1,12 +1,9 @@
 {
   "compilerOptions": {
+    "composite": true,
```

Comment:
Add `composite` based on this commit: https://github.com/vitejs/vite/pull/15913/files#diff-4aee86d836cbe0aac0d45ef61a8ca540455ef2a7ee667abb3dc1fdac0956b6a5